packages/connect-popup/src/view/selectDevice.ts

```diff
@@ -32,16 +30,19 @@ const initWebUsbButton = (showLoader: boolean) => {
             await window.navigator.usb.requestDevice({
                 filters: TREZOR_USB_DESCRIPTORS,
             });
-            const { iframe } = getState();
-            if (!iframe) {
-                throw ERRORS.TypedError('Popup_ConnectionMissing');
-            }
-            iframe.postMessage({
-                event: UI_EVENT,
-                type: TRANSPORT.REQUEST_DEVICE,
-            });
-            if (showLoader) {
-                showView('loader');
+
+            const { iframe, core } = getState();
+
+            if (iframe) {
+                iframe.postMessage({
+                    event: UI_EVENT,
+                    type: TRANSPORT.REQUEST_DEVICE,
+                });
+                if (showLoader) {
+                    showView('loader');
+                }
+            } else if (core) {
+                core.enumerate();
```

Comment:
If there is no iframe nor core there is no error here as there was before?

Reply:
yeah, maybe I put the error back